### DISCUSSION TITLE
A problem about TemporalwiseAttention

### DISCUSSION THREAD

**Discussion by KarryChiang** [fangwei123456/SpikingFlow#272] [General]
Hello! I encountered a problem when using the module **TemporalWiseAttention** to run the code. The version is **spikingjelly 0.0.0.0.13-py10** now. I previously used the module called **MultiStepStepTemporalWiseAttention** on **spikingjelly 0.0.0.0.11-py10** without any problem. Because I have not got previous version of this code now, there is no way to compare. So I would like to know what is the problem with the error shown in the figure? I will be grateful if you could give me some suggestions, may you have a nice day!
<img width="955" alt="image" src="https://user-images.githubusercontent.com/77139502/192964408-22aa5329-5072-4336-9b90-1cda87a1cc1b.png">

**Comment by fangwei123456** [comment]
Can you privide the minimal codes to reproduce this error?
